$OpenBSD: patch-src_floating_c,v 1.21 2015/12/18 15:53:09 dcoppa Exp $

Fix crash when trying to split and float a dock container.

--- src/floating.c.orig	Wed Sep 30 08:55:10 2015
+++ src/floating.c	Fri Dec 18 13:58:58 2015
@@ -108,7 +108,7 @@ void floating_check_size(Con *floating_con) {
 void floating_enable(Con *con, bool automatic) {
     bool set_focus = (con == focused);
 
-    if (con->parent && con->parent->type == CT_DOCKAREA) {
+    if (con_is_docked(con)) {
         LOG("Container is a dock window, not enabling floating mode.\n");
         return;
     }
